integrators/integrators.hpp

```diff
@@ -211,21 +215,12 @@ class AdaptiveStepSizeIntegrator : public Integrator<ODE_> {
 
   virtual void WriteToMessage(
       not_null<serialization::AdaptiveStepSizeIntegrator*> message) const = 0;
+  template<typename S = typename ODE::SystemState,
+           typename = std::enable_if_t<base::is_serializable_v<S>>>
```

Comment:
Why?

Reply:
Changed to allow deserialization of integrators for non-serializable frames (both ASS and FSS).